test/distributed/_tensor/test_dtensor_log.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# Owner(s): ["oncall: distributed"]
+import functools
+import os
+import unittest
+
+import torch.distributed as dist
+from torch._dynamo.test_case import run_tests
+from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
+from torch.testing._internal.logging_utils import LoggingTestCase
+
+requires_distributed = functools.partial(
+    unittest.skipIf, not dist.is_available(), "requires distributed"
+)
+
+
+@skip_if_lt_x_gpu(2)
+class DTensorLogTest(LoggingTestCase):
```

Comment:
Do we expect to add more test cases for this? If not we can consider merging this into `test_dtensor.py`

Reply:
Merged